Lab 3.ipynb

```diff
@@ -0,0 +1,331 @@
+      "text/plain": [
+       "<Figure size 1200x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "d = np.append(stats.norm.rvs(loc = 12., scale = 0.4, size = 100000), [10., 10.3, 2.1, 0., 0., 15.6, 22.3, 12.7])\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "ax.hist(d, 150, density=True)\n",
+    "ax.tick_params(labelsize = 24)\n",
+    "ax.set_ylim([10e-6, 2])\n",
+    "ax.set_yscale('log')\n",
+    "ax.set_xlabel(\"Temperature\")\n",
+    "ax.set_ylabel(\"Number of Readings (density)\")\n",
+    "x = np.linspace(0, 25,1000)\n",
+    "ax.plot(x,stats.norm.pdf(x, loc = 12, scale = 0.4),linewidth = 7,alpha = 0.7)\n",
+    "plt.show()"
```

Comment:
Explain what this plot is showing and what data points are bad. Would also be good to include a legend to differentiate between the colors